### ISSUE TITLE
Tap gestures not working

### ISSUE THREAD

**Issue by alexstanbury** [Flair-Dev/react-native-awesome-gallery#41]
Hi, first of all, thanks for making the library, it's very handy.

I'm using 0.3.4, unfortunately tap and double tap gestures are not firing on either iOS or Android. 

Example code:

```
 <Gallery
        disableTransitionOnScaledImage
        ref={galleryRef}
        data={images}
        onDoubleTap={() => {
          //doesn't work
          console.log('double tap');
        }}
        onTap={() => {
          //doesn't work
          console.log('tap');
        }}
         onScaleChange={() => {
          //works
          console.log('scale');
        }}
      />

```

**Comment by pavelbabenko** [comment]
@alexstanbury Which version of gesture handler do you have?

**Comment by alexstanbury** [comment]
"react-native-gesture-handler": "^2.3.0"

**Comment by alexstanbury** [comment]
Just tested with a fresh react native project and still getting the same issue. It looks as though the pan start event is taking precedence over any tap events, as a guess you might need to add an activation threshold for the pan event activation.

**Comment by pavelbabenko** [comment]
Fixed in [v0.3.5](https://github.com/Flair-Dev/react-native-awesome-gallery/releases/tag/v0.3.5)

**Comment by alexstanbury** [comment]
Thank you!
